app/rest/rest/src/main/java/io/syndesis/rest/v1/handler/integration/IntegrationHandler.java

```diff
@@ -200,7 +194,15 @@ public Integration create(@Context SecurityContext sec, @ConvertGroup(from = Def
             .stepsDone(new ArrayList<>())
             .build();
 
-        return Creator.super.create(sec, updatedIntegration);
+        updatedIntegration = Creator.super.create(sec, updatedIntegration);
+
+        IntegrationRevision revision = IntegrationRevision
+            .createNewRevision(updatedIntegration)
+            .withCurrentState(IntegrationRevisionState.Draft)
+            .withTargetState(IntegrationRevisionState.Active);
+
+        getDataManager().create(revision);
```

Comment:
We should only be creating revisions/deployments when the user requests it to be published.